GUI/test/javafxgui/controller/GUIAppControllerTest.java

```diff
@@ -88,13 +96,10 @@ public void tellGUIDisplayToDisableBoard() {
                 X, X, O
         };
 
-        GUIGameMakerFake gameMaker = new GUIGameMakerFake(new GUIPlayerFactory());
-        gameMaker.setDummyBoard(arrayToList(currentBoard));
+        fakeGameMaker.setDummyBoard(arrayToList(currentBoard));
 
-        Controller controller = new TTTController(gameViewSpy, gameMaker);
-        controller.initializeGame(GUI_HVH, DEFAULT_BOARD_DIMENSION);
-        controller.displayGameLayout();
-        controller.displayResult();
+        GUIAppController controller = new GUIAppController(fakeGameMaker, gameViewSpy);
+        controller.startGame(GUI_HVH, DEFAULT_BOARD_DIMENSION);
         assertEquals(true, gameViewSpy.hasBoardBeenDisabled());
```

Comment:
consider using hamcrest matchers to improve the readability of these asserts. Alternatively, assertTrue would read better too I think.